test/test_nestedtensor.py

```diff
@@ -527,6 +535,25 @@ def test_data_ptr(getter):
                     self.assertIs(torch.int32, nt2.to(dtype=torch.int32).dtype)
                     self.assertEqual(nt2.device, nt2.to(dtype=torch.int32).device)
 
+        # Jagged to strided (should work)
+        if layout == torch.jagged:
+            strided_nt = torch.ops.aten._to_copy(nt, layout=torch.strided)
+            self.assertIs(strided_nt.layout, torch.strided)
+            self.assertEqual(strided_nt.device, nt.device)
+            self.assertEqual(strided_nt.size(2), nt.size(2))
+            self.assertEqual(strided_nt.unbind(), nt.unbind())
```

Comment:
nit: (here and below for the torch.strided case) might be worth also checking that the two NTs don't reference the same storage since the func is supposed to copy